Adyen/Service/PosPaymentLocalApi.cs

```diff
@@ -92,11 +62,10 @@ public SaleToPOIResponse TerminalApiLocal(SaleToPOIMessage saleToPoiRequest, Enc
         /// <param name="notification"></param>
         /// <param name="encryptionCredentialDetails"></param>
         /// <returns></returns>
+        [Obsolete("This in person payment class in deprecated and will be remove in the next major, please refer to TerminalLocalApi.cs")]
         public string DecryptNotification(string notification, EncryptionCredentialDetails encryptionCredentialDetails)
         {
-            var saleToPoiMessageSecured = _saleToPoiMessageSecuredSerializer.Deserialize(notification);
-            var decryptNotification = _messageSecuredEncryptor.Decrypt(saleToPoiMessageSecured, encryptionCredentialDetails);
-            return decryptNotification;
+            _terminalLocalApi.DecryptNotification(notification, encryptionCredentialDetails);
```

Comment:
Like github actions suggests
```suggestion
            return _terminalLocalApi.DecryptNotification(notification, encryptionCredentialDetails);
```